src/frontend/get-github-setup.ts

```diff
@@ -1,23 +1,22 @@
 import { jiraDomainOptions } from "./validations";
-import { NextFunction, Request, Response } from "express";
-import { getGitHubConfigurationUrl } from "../util/getUrl";
+import { Request, Response } from "express";
+import { getJiraMarketplaceUrl } from '../util/getUrl';
 
 /*
 When this request is made: Installing from Jira Marketplace - GitHub org does not have Jira installed.
 Redirects users back to github/configuration to install their Jira instance in GitHub org/s.
 If the installation was done from Jira Marketplace, the app is already installed.
 */
-export default (req: Request, res: Response, next: NextFunction): void => {
+export default (req: Request, res: Response): void => {
 	req.log.info("Received get github setup page request");
-	const { host: githubHost, session: { jwt, jiraHost } } = req;
-	if (jiraHost && jwt) {
-		return res.redirect(getGitHubConfigurationUrl({ githubHost, jwt, jiraHost }));
+
+	if (req.session.jiraHost) {
+		res.redirect(getJiraMarketplaceUrl(req.session.jiraHost));
```

Comment:
something that would be nice here is if they have a jiraHost (or after they've inputted it in github/setup), that there would be an API on our end to confirm if the host already has the app installed.  If they do, go directly to the manage app section, if not, bring them to the marketplace URL to install it.

Reply:
As mentioned in the PR description, there is already a ticket (#620) in the backlog to follow up on this. I think we should decouple this from the fix for the broken flow and get this in as soon as possible.